tools/seed-kopis/reschedule-dates.py

```
@@ -0,0 +1,198 @@
+def parse_dt(s: str) -> datetime:
+    return datetime.strptime(s, DT)
+
+
+def main() -> None:
+    ap = argparse.ArgumentParser()
+    ap.add_argument("path")
+    ap.add_argument("--anchor", default=None, help="기준일 YYYY-MM-DD (기본: 오늘)")
+    args = ap.parse_args()
+
+    anchor = (
+        datetime.strptime(args.anchor, D) if args.anchor else datetime.now()
```

Ruff (0.15.15)

[warning] Naive datetime constructed using `datetime.datetime.strptime()` without %z

(DTZ007)